api/pyproject.toml

```diff
@@ -4,16 +4,16 @@ build-backend = "poetry.masonry.api"
 
 [tool.poetry]
 name = "doctr-api"
-version = "0.5.2a0"
+version = "1.0.1a0"
 description = "Backend template for your OCR API with docTR"
 authors = ["Mindee <contact@mindee.com>"]
 license = "Apache-2.0"
 
 [tool.poetry.dependencies]
-python = ">=3.8,<3.11"
+python = ">=3.8.2,<3.11"
```

Comment:
Could you add a comment to specify the reason of the minor version? (much easier to reassess later when we have a comment saying why)

Reply:
Yes good idea ! added it.